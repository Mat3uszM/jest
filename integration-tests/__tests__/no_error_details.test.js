/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */
'use strict';

const runJest = require('../runJest');
const {extractSummary} = require('../Utils');

describe('--noErrorDetails switch', () => {
  test('suppresses printing the cause of a failed test', () => {
    const {stderr} = runJest('no-error-details', [
      'test_error.test.js',
      '--noErrorDetails',
    ]);
    expect(extractSummary(stderr).rest).toMatchSnapshot();
  });
});
