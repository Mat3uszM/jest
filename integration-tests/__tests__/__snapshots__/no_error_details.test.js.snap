// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--noErrorDetails switch suppresses printing the cause of a failed test 1`] = `
"FAIL __tests__/test_error.test.js
  âœ• failing test

"
`;
